pages/1.11/release-notes/1.11.1/index.md

```diff
@@ -0,0 +1,111 @@
+
+### Storage
+- DC/OS Storage Service 0.1 (beta) - DSS users will be able to dynamically create volumes based upon profiles or policies to fine-tune their applications' storage requirements. This feature leverages the industry-standard Container Storage Interface (CSI) to streamline the development of storage features in DC/OS by Mesosphere and our community and partner ecosystems. [View the documentation](https://docs.mesosphere.com/services/beta-storage/0.1.0-beta/).[beta type="inline" size="small" /][enterprise type="inline" size="small" /]
+- Pods now support persistent volumes. [View the documentation](/1.11/deploying-services/pods).[beta type="inline" size="small" /]
+
+**Note:** Because these storage features are beta in 1.11, they must be explicitly enabled. Beta features are not recommended for production usage, but are a good indication of the direction the project is headed.
+
+### Updated DC/OS Data Services
+- TLS encryption for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS is now supported.
+- Fault domain awareness for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic and DC/OS HDFS. Use fault domain awareness to make your services highly available and to allow for increased capacity when needed.
+- New API endpoint to pause a node for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS. Use this endpoint to relaunch a node in an idle command state for debugging purposes.
+- New DC/OS Kafka ZooKeeper service. [View the documentation](/services/kafka-zookeeper).
+- You can now select a DC/OS data service version from a dropdown menu in the DC/OS UI.
+- Improved scalability for all DC/OS data services.
+
+## <a name="issues-fixed"></a>Improvements and Issues Fixed in DC/OS 1.11.1 Release
+
+- INFINITY-3331 - Fixed cleaning up other framework's volumes.
+- DCOS_OSS-2292 - Fixed crashes on empty lines for DC/OS CLI.
+- DCOS_OSS-2255 - Fixed dcos-installer-ui package failure to build.
+- DCOS_OSS-2247 - Fixed fatal error on checks that exceeded timeout on dcos-diagnostic service.
+- DCOS_OSS-2223 - Added encoders to support package registry. Fixed comsos to initialize empty objects for optional json elements from json schema.
+- DCOS_OSS-2210 - Ensured that cache timestamps are well-formed.
+- DCOS_OSS-2184 - Removed an invalid dns record for marathon.mesos.
+- DCOS_OSS-2087 - Improved readability on user facing messages during service uninstallation.
+- DCOS_OSS-1759 - Updated package-manager.yaml to fix the schema error in packaage management API.
+- DCOS-21305 - Added a feature on DC/OS to support the minimal requested version when installing universe packages(cannot install a package which requires 1.11 on 1.10).
+- DCOS-21337 - Added linearBackoff retry to the Mesos stream.
+- DCOS-21266 - Fixed file navigation in the UI.
+- DCOS-21128 - Fixed uninstalling failing service breaks the services tab.
```

Comment:
DC/OS UI: Fixed a scenario in which the services tab crashed after uninstalling a service.

Reply:
Done.